packages/tasks/src/tasks/text-generation/data.ts

```diff
@@ -76,7 +76,7 @@ const taskData: TaskDataCustom = {
 		},
 		{
 			description: "A very powerful model with reasoning capabilities.",
-			id: "PowerInfer/SmallThinker-3B-Preview",
+			id: "qihoo360/TinyR1-32B-Preview",
```

Comment:
bit dodgy - I'd skip this one and maybe keep the original one or use 

```suggestion
			id: "simplescaling/s1.1-32B",
```